src/app/container/refresh-wizard/refresh-wizard.component.html

```diff
@@ -0,0 +1,72 @@
+<div class="modal-body">
+  <p>
+    Select a Quay.io namespace and then click the sync icon next to a repository to automatically register or sync the tool on Dockstore.
+    Requires a Quay.io account linked on Dockstore and also Git build triggers to be present on the repository.
+  </p>
+  <app-loading [loading]="loading$ | async">
+    <mat-card class="alert alert-danger" role="alert" *ngIf="error$ | async as error">
+      <button type="button" class="close" data-dismiss="alert" (click)="clearError()">
+        &times;
+      </button>
+      <p>
+        <mat-icon>warning</mat-icon>
+        The webservice encountered an error.
+      </p>
+      <p class="error-output">
+        {{
+          '[HTTP ' +
+            error.status +
+            '] ' +
+            error.statusText +
+            ': ' +
+            (error.error && error.error.message ? error.error.message : error.error)
+        }}
+      </p>
+    </mat-card>
+    <div *ngIf="organizations$ | async as organizations">
+      <div *ngIf="organizations === undefined">
+        Encountering an issue with the website.
+      </div>
+      <div *ngIf="organizations === null">
+        Problems getting organizations.
+      </div>
+      <div *ngIf="organizations && organizations.length === 0">
+        Could not get any organizations, try relinking the Quay.io token in the accounts page.
+      </div>
+      <div *ngIf="organizations && organizations.length > 0">
+        <mat-form-field class="w-100">
+          <mat-label>Select namespace</mat-label>
+          <mat-select (selectionChange)="selectOrganization($event.value)">
+            <mat-option *ngFor="let organization of organizations" [value]="organization">
+              {{ organization }}
+            </mat-option>
+          </mat-select>
+        </mat-form-field>
+        <div *ngIf="repositories$ | async as repositories">
+          <div *ngIf="repositories">
+            <div *ngFor="let repository of repositories">
+              <button
+                mat-icon-button
+                color="primary"
+                aria-label="Refresh button"
+                (click)="refreshRepository(repository)"
+                [disabled]="(loading$ | async) || (repositoryLoading$ | async)"
+                matTooltip="Refresh on Dockstore"
+              >
+                <mat-icon>sync</mat-icon>
```

Comment:
The sync icon doesn't have the same state information that exists for workflows in the form of the slider. What was the reasoning for using the refresh icon over the slider?

Reply:
workflow deregisters, tool doesn't.  this refreshes an existing one or adds a new one.

Reply:
But if it refreshes an existing one, that doesn't make sense since it is a register tool modal and not a refresh tool modal.

Reply:
this is actually called the refresh-wizard component.